core/src/commonMain/kotlin/com/powersync/connectors/PowerSyncBackendConnector.kt

```diff
@@ -69,6 +70,7 @@ public abstract class PowerSyncBackendConnector {
      *
      * This token is kept for the duration of a sync connection.
      */
+    @SuspendInterop.Disabled
```

Comment:
Why do we want to disable the suspend functionality of these functions?

Reply:
Here is what we received from the audit report:

`At the moment, SKIE’s suspend bridging introduces a prefix of two underscores to the original
suspend method. That means that users have to implement __fetchCredentials() and
__uploadData(database:) in their code. Since these methods aren’t supposed to be called
from Swift, we recommend turning SKIE off for the PowerSyncBackendConnector class.
SKIE currently can’t bridge the implementation back to Kotlin, so it doesn’t bring the
cancellation/cooperative concurrency benefit. You can disable SKIE’s suspend feature for this class
through either annotations or Gradle configuration in the powersyncswift module.`